src/cloud/digitalocean/provisioner.ts

```diff
@@ -104,8 +105,13 @@ class Provisioner {
    * @param {String} name of VM to create
    * @return {Promise.<Object>}
```

Comment:
Let's quickly describe the format of the return type while we're at it...is it just:
```js
{
  droplet: <droplet id, as a string>
}
```
?